### PR TITLE
[SP-5244] Add support for consent language

### PR DESCRIPTION
**Pull request by vilasma**
1. In Web message case we are sending **consentLanguage** to sendConsent API call, consentLanguage is captured from action event.
1. In Native message case we are sending **Locale** value as consentLanguage value to sendConsent API call.

### CODE REVIEW

File: ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift [hunk 1]
```diff
@@ -182,7 +182,9 @@ class SourcePointClient: SourcePointProtocol {
                 requestUUID: requestUUID,
                 pmSaveAndExitVariables: pmPayload,
                 meta: meta,
-                publisherData: action.publisherData
+                publisherData: action.publisherData,
+                consentLanguage: (action.consentLanguage != nil) ?
+                    action.consentLanguage : Locale.preferredLanguages[0].uppercased()
```
Comment (andresilveirah) [inline ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift:187]: @vilasma I think we shouldn't put the responsibility to fill in the default language `Locale.preferredLanguages[0].uppercased()` in the `SPClient`. 
This should be set as soon as the `Action` is created.

Comment (vilasma) [reply ConsentViewController/Classes/SourcePointClient/SourcePointClient.swift]: @andresilveirah added Locale value in once action is triggered from native message

File: ConsentViewController/Classes/GDPRAction.swift [hunk 2]
```diff
@@ -33,7 +33,7 @@ import Foundation
 @objcMembers public class GDPRAction: NSObject {
     public let type: GDPRActionType
     public let id: String?
-    public let consentLanguage: String?
+    public var consentLanguage: String?
```
Comment (andresilveirah) [inline ConsentViewController/Classes/GDPRAction.swift:36]: @vilasma does `consentLanguage` need to be a `var`? Are we reassigning it at some point?

Comment (vilasma) [reply ConsentViewController/Classes/GDPRAction.swift:36]: Yes @andresilveirah, In native message case we are assigning the `Locale` value to `consentLanguage`.

Comment (vilasma) [reply ConsentViewController/Classes/GDPRAction.swift]: I have updated consentLanguage property type to let

**Pull request merged**